src/components/Table/Table.component.tsx

```
@@ -0,0 +1,86 @@
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {data
+                            .slice(
+                                PAGE * ROWS_PER_PAGE,
+                                PAGE * ROWS_PER_PAGE + ROWS_PER_PAGE,
+                            )
+                            .map((row, index) => (
+                                <TableRow
+                                    hover
+                                    role="checkbox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<input type="checkbox">

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)